### PR TITLE
docs: convert dependency-injection-in-action code example

### PR DESCRIPTION
**Pull request by MarkTechson**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Current this code is ngModule based

## What is the new behavior?
The code will be based on standalone component arch

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by twerske — APPROVED** [review]

File: aio/content/examples/dependency-injection-in-action/src/app/app.module.ts
Comment (jessicajaniuk) [inline aio/content/examples/dependency-injection-in-action/src/app/app.module.ts:1]: I'm guessing this module file is no longer needed. Did you want to delete it?

File: aio/content/examples/dependency-injection-in-action/src/app/app-routing.module.ts
Comment (jessicajaniuk) [inline aio/content/examples/dependency-injection-in-action/src/app/app-routing.module.ts:1]: Is this file necessary?

**Review by jessicajaniuk — COMMENTED** [review]
There's a bunch of empty files here, some look to be modules that probably were intended to be deleted, but others that seem like there should be content. Is that intentional?

**Comment by MarkTechson** [comment]
@jessicajaniuk something strange happened with my original commit. I've fixed it up and got rid of the empty files and the unused modules.

**Review by jessicajaniuk — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 429b76eacf62dcec445f17fa9c693f87a74e3eb4.
